aten/src/ATen/native/Convolution.cpp

```diff
@@ -696,7 +710,11 @@ at::Tensor _convolution(
     }
 #endif
   } else if (input.device().type() == c10::DeviceType::CPU || input.device().type() == c10::DeviceType::CUDA) {
-    if (params.use_cpu_depthwise3x3_winograd(input, weight)) {
+    if (params.use_xnnpack(input, weight, bias)) {
```

Comment:
It seems that for depthwise 3x3 they have ARM_NEON implementation of winowgrad. https://github.com/pytorch/pytorch/blob/master/aten/src/ATen/native/Convolution.cpp#L128. Do you know if xnnpack is still faster for depthwise compared to that implementation? Not sure if you saw this while doing experiments with mobilenet and resnet.

Reply:
Looks like you added that.

Reply:
Like everything performance I think the best solution would be not to make any assumptions and instead benchmark.  I will conduct a bunch of benchmarks and get back.